src/texture_stage.cpp

```diff
@@ -62,7 +62,7 @@ void TextureStage::Commit(uint32_t memory_dma_offset, uint32_t palette_dma_offse
   // NV097_SET_TEXTURE_OFFSET
   p = pb_push2(p, NV20_TCL_PRIMITIVE_3D_TX_OFFSET(stage_), texture_addr, format);
 
-  uint32_t pitch_param = (format_.xbox_bpp * width_) << 16;
+  uint32_t pitch_param = uint32_t(format_.xbox_bpp * width_) << 16;
```

Comment:
What happens with a width of 1 on hardware?

Reply:
Good question. I don't know it that is a valid width, however I saw xbox accepting 1px height, so it could be. 
If we think in terms of bytes size we can only send a 1 or 0 i guess. However maybe it could be possible to send bit size as well? Maybe using bit size and shifting it with 13 instead of 16? I will try it on my hardware, see what happens.

Reply:
I tried it but it does not seem to work with either way. It halts the system. Maybe it is not a valid width for this texture format.

Reply:
That makes sense, and it's good to know it's not doing any interesting graceful handling of malformed DXT1's - thanks for checking!

Reply:
Xemu does not handle it gracefully though. It gives a different image result every time I run it.

Reply:
There are definitely a handful of places where xemu is more lenient than the hardware; I'd suggest filing a bug against xemu describing the difference. In practice it'll presumably never be a problem outside of homebrew development since it'd fail consistently on hardware, so it may or may not be worth adding an assert or exception case to xemu.

Reply:
Okay, I will do that in conjunction with the problem of 1px height, because the problem there is the opposite. Xemu asserts, and the hardware is fine with it. 
The next PR will add custom resolution support for textures, so there we could introduce tests for these cases.